packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -120,6 +87,62 @@ type BaseDesignTokens = {
 
 type BaseColorSystem = Pick<BaseDesignTokens, 'palette' | 'shadowRing' | 'shadowChannel'>;
 
+const createLightModeVariantVariables = (color: ColorPaletteProp) => ({
```

Comment:
There is no change on these functions, just moved there to prevent dependency cycle.